recipes/ipycache/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.1.4" %}
+
+package:
+  name: ipycache
+  version: {{ version }}
+
+source:
+  fn: ipycache-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/i/ipycache/ipycache-{{ version }}.zip
+  md5: c78c6094ffab45b0e33c2f556705fa11
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - ipython
+
+test:  
+  commands:
+    - git clone --depth=50 https://github.com/rossant/ipycache.git source
+    - cd source 
+    - nosetests 
+    - python ipynb_runner.py -v -s examples/example.ipynb  # [not win]
+    - python ipynb_runner.py -v -s examples/example_outputs.ipynb  # [not win]
+    - cd ..
+  
+  requires:
+    - nose
+    - numpy
+    - matplotlib
+    - ipython
+    - nbformat
+    - ipykernel
+
+about:
+  home: http://packages.python.org/ipycache
+  license: BSD License
```

Comment:
We may have a problem here. The PyPI distro (made by the author) states just OSI approved BSD, while the github repo contains author-written license which I do not recognize but is a fairly liberal short license similar to BSD. I do not feel competent to judge that. Since we are basing on the *author-released* pypi package I would argue we can go with the license he specified for pypi?